zerver/tests/test_auth_backends.py

```diff
@@ -435,6 +421,11 @@ def setUp(self) -> None:
         from social_core.backends.utils import load_backends
         load_backends(settings.AUTHENTICATION_BACKENDS, force_load=True)
 
+    def registerExtraEndpoints(self,
```

Comment:
This should probably be underscore-named, not camelCase, since it's a function, not a class.

Reply:
Done, in a separate commit since part of it was already merged! :)